gammapy/data/event_list.py

```diff
@@ -191,9 +198,7 @@ def from_stack(cls, event_lists, **kwargs):
         Parameters
         ----------
         event_lists : list
-            List of `~gammapy.data.EventList` to stack.
-        **kwargs : dict, optional
-            Keyword arguments passed to `~astropy.table.vstack`.
+            List of `~gammapy.data.EventList` to stack
         """
         tables = [_.table for _ in event_lists]
         stacked_table = vstack_tables(tables, **kwargs)
```

Comment:
In this function, one would need to pass some meta data too. 
As far as I can, there was no real check on the `table.meta`. 
I assume that the correct behavior is to check that specific meta contents are consistent. 
Maybe simply raise a warning that `meta` will be empty here.

Reply:
`vstack` has a `metadata_conflicts` argument with default set to `warn`.
So it should be handled within that?

I added a warning for `meta` being empty

Reply:
Maybe add a TODO as well